.github/workflows/certora.yml

```diff
@@ -26,11 +26,11 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      
+
       - uses: actions/setup-java@v3
         with:
-          distribution: 'zulu'
-          java-version: '11'
+          distribution: "zulu"
+          java-version: "11"
```

Comment:
We might want to roll back this change as well as a bit weird to have it here and actually doesn't change anything on how it works.

Reply:
Yes it's not related to this PR, it was introduced automatically by the formatter (prettier in this case)